jarbas/core/tests/test_reimbursement_model.py

```diff
@@ -26,6 +26,17 @@ def test_unique_document_id(self):
         with self.assertRaises(IntegrityError):
             Reimbursement.objects.create(**self.data)
 
+    def test_last_update(self):
+        reimbursement = Reimbursement.objects.create(**self.data)
```

Comment:
I agree with @berinhard ... There's some options for fixture replacement... I would try [mixer](https://github.com/klen/mixer)